### PR TITLE
Create Driver ShuffleBoard

### LINKED ISSUE #81: Code Shuffle board layout
Like last year, we need to code a shuffleboard layout for Auto and TeleOp

### PR DESCRIPTION
**Pull request by legoguy1000**
closes #81

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/elevator_wrist/ElevatorWrist.java [hunk 1]
```diff
@@ -48,6 +52,16 @@ public class ElevatorWrist extends SubsystemBase {
 
     private InterpolatingDoubleTreeMap radiusToAngle = new InterpolatingDoubleTreeMap();
 
+    private GenericEntry wristAngle = RobotContainer.mainDriverTab.add("Wrist Angle", 0)
+        .withWidget(BuiltInWidgets.kDial).withProperties(Map.of("Min", 0, "Max", 360))
```
Comment (wilsonwatson) [inline src/main/java/frc/robot/subsystems/elevator_wrist/ElevatorWrist.java:56]: Min/Max should be something like (-180, 180). 0 is flat, so we do go negative during amp/trap scoring.

**Review by wilsonwatson — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/elevator_wrist/ElevatorWrist.java (hunk 1 above)
Comment (legoguy1000) [reply src/main/java/frc/robot/subsystems/elevator_wrist/ElevatorWrist.java:56]: done

**Comment by legoguy1000** [comment]
Shuffle Board
image
Elastic Dashboard (https://github.com/Gold872/elastic-dashboard)
image

**Review by owenflatman — DISMISSED** [review]

**Comment by legoguy1000** [comment]
image

**Comment by legoguy1000** [comment]
Elastic dashboard configured via API. Have Note wil actually flash between green and blue like LEDs when we have a note

**Review by owenflatman — APPROVED** [review]

**Pull request merged**